packages/shared/src/forms/__tests__/__snapshots__/Checkbox.test.js.snap

```diff
@@ -36,15 +36,21 @@ exports[`checked 1`] = `
       style={
         Object {
           "alignItems": "flex-end",
-          "flex": 1,
           "justifyContent": "center",
         }
       }
     >
-      <Icon
-        color="#00a991"
-        name="check"
-        size={26}
+      <TextIcon
+        code="S"
```

Comment:
Maybe we should also test that it renders correctly on android since there now are different icons?